tools/c7n_mailer/tests/common.py

```diff
@@ -44,7 +44,7 @@
     'role': 'arn:aws:iam::xxxx:role/cloudcustodian-mailer',
     'ldap_uid_tags': ['CreatorName', 'Owner'],
     'templates_folders': [os.path.abspath(os.path.dirname(__file__)),
-                          os.path.abspath('/')],
+                          os.path.abspath('/'), ''],
```

Comment:
the posixpath.join and os.path.join behavior differential causes us to do a look up on files like `C:\\C:/foo/bar/example.jinja`. We pass a empty path here to ensure that we are able to do an abs path join